frontend/components/card/change-account-info/CardChangeAccountInfo.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="flex justify-center h-min card-style md:justify-start">
+    <div class="flex flex-col p-2 md:flex-row md:grow sm:px-5 sm:py-3 h-fit">
+      <div v-if="infoType === 'email'" class="w-full">
+        <CardChangeAccountInfoEmail />
+        <div class="flex w-full pt-4">
+          <div class="w-1/3">
+            <BtnCaptcha />
+          </div>
+          <div class="w-1/3">
+            <BtnChangeDetails info-type="email" />
+          </div>
+        </div>
+      </div>
+      <div v-if="infoType === 'username'" class="w-full">
+        <CardChangeAccountInfoUsername />
+        <div class="flex w-full pt-4">
+          <div class="w-1/3">
+            <BtnCaptcha />
+          </div>
+          <BtnChangeDetails info-type="username" />
+          <div class="w-1/3"></div>
+        </div>
+      </div>
+      <div v-if="infoType === 'password'" class="w-full">
+        <CardChangeAccountInfoPassword />
+        <div class="flex w-full pt-4">
+          <div class="w-1/3">
+            <BtnCaptcha />
+          </div>
+          <div class="w-1/3">
+            <BtnChangeDetails info-type="password" />
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  header: String;
+  description: String;
+  infoType: "email" | "username" | "password";
+  ctaLabel: String;
```

Comment:
Hi @RyanPaulGannon, where are `header` `description` and `ctaLabel` props being use ?

Reply:
Hey @rigojr, I've been making some adjustments recently, just haven't had the time to finish and push the update just yet :)

Reply:
Looking forward to it, @RyanPaulGannon!